packages/compass-schema/src/components/promo-banner.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {
+  css,
+  cx,
+  Link,
+  palette,
+  spacing,
+  useDarkMode,
+  Body,
+} from '@mongodb-js/compass-components';
+import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
+
+const { track } = createLoggerAndTelemetry('COMPASS-SCHEMA-UI');
+
+const bannerBodyStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  gap: spacing[3],
+  paddingTop: spacing[2],
+  paddingBottom: spacing[2],
+  paddingLeft: spacing[3],
+  paddingRight: spacing[3],
+  borderRadius: '12px',
+  marginLeft: spacing[3],
+  marginRight: spacing[3],
+  '&:not(:last-child)': {
+    marginBottom: spacing[4],
+  },
+});
+
+const bannerBodyLightModeStyles = css({
+  backgroundColor: palette.gray.light3,
+  boxShadow: `inset 0 0 0 1px ${palette.gray.light2}`,
+});
+
+const bannerBodyDarkModeStyles = css({
+  backgroundColor: palette.gray.dark3,
+  boxShadow: `inset 0 0 0 1px ${palette.gray.dark2}`,
+});
+
+const bannerIconStyles = css({
+  width: '32px',
+  height: '32px',
```

Comment:
Is this not spacing as it's paired with the `svg` width/height below? Might be nice to have one variable that's shared in that case. Small thing, totally fine as is, feel free to ignore.
```suggestion
  width: spacing[5],
  height: spacing[5],
```

Reply:
Yeah, only for that reason, let me update in both places actually